tst/EnergyPlus/unit/InputProcessor.unit.cc

```diff
@@ -74,6 +74,25 @@ namespace EnergyPlus {
 
 		}
 
+		TEST_F(InputProcessorFixture, MisleadingIDDWarningTest)
+		{
+			bool errors_found = false;
+
+			std::string const idf_contents = delimited_string({
+				"Timestep, 4;"
+			});
+
+			ASSERT_FALSE( process_idf( idf_contents, errors_found ) );
+			EXPECT_TRUE( compare_err_stream( "", true ) );
+
+			std::string saveMatchVersion = DataStringGlobals::MatchVersion;
+			DataStringGlobals::MatchVersion = "2.0";
+			ASSERT_FALSE(process_idf(idf_contents, errors_found ));
+			EXPECT_TRUE( compare_err_stream( "   ** Severe  ** IP: Possible incorrect IDD File\r\n   **   ~~~   ** " + DataStringGlobals::IDDVerString +" not the same as expected =\"2.0\"\r\n", true ) );
```

Comment:
Ah -` delimited_string()`of course - thanks!  Regarding what's getting tested, that's up to the reviewer.  Maybe it would be better to move this to a checkIDDversion function.  Then it can be tested directly and it can be called from both process_idf and `EnergyPlusFixture::process_idf()` function and `ProcessInput()`.